stdlib/tkinter/ttk.pyi

```diff
@@ -618,9 +618,9 @@ class Progressbar(Widget):
     @overload
     def configure(self, cnf: str) -> tuple[str, str, str, Any, Any]: ...
     config = configure
-    def start(self, interval: Any | None = ...): ...
-    def step(self, amount: Any | None = ...): ...
-    def stop(self): ...
+    def start(self, interval: Literal["cancel", "idle", "info"] | int | None = ...) -> None: ...
```

Comment:
`cancel` and `info` shouldn't be allowed. The fact that the error message mentions them is an accident of implementation, not a feature. (It gets passed to `after` command, but `after cancel` and `after info` don't schedule a new timeout.)

```suggestion
    def start(self, interval: Literal["idle"] | int | None = ...) -> None: ...
```